app/Http/Controllers/Admin/CategoryController.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use Illuminate\Http\Request;
+use App\Http\Controllers\Controller;
+use App\Models\Category;
+use Session;
+use App\Http\Requests\CategoryRequest;
+
+class CategoryController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $data = Category::orderBy('id', 'DESC')->paginate(config('constant.LIMIT_PAGE_ADMIN'));
+        return view('admin.categories.list', compact('data', $data));
```

Comment:
the view file name should match with the controller method name. please change to `admin.categories.index`